pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -462,6 +464,31 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 			newMounterErr)
 	}
 
+	deviceMountPath := ""
+	if attachablePlugin != nil {
+		attacher, newAttacherErr := attachablePlugin.NewAttacher()
+		if newAttacherErr != nil {
+			return nil, fmt.Errorf(
+				"AttachableVolumePlugin.NewAttacher failed for volume %q (spec.Name: %q) pod %q (UID: %q) with: %v",
+				uniqueVolumeName,
+				volumeSpec.Name(),
+				volume.podName,
+				pod.UID,
+				newAttacherErr)
+		}
+
+		var getDeviceMountPathErr error
+		deviceMountPath, getDeviceMountPathErr = attacher.GetDeviceMountPath(volumeSpec)
```

Comment:
For flex volume, what this will return?

Reply:
It returns the mount path where the volume is expected to be mounted.